frontend/components/DataError/DataError.tsx

```diff
@@ -15,6 +15,7 @@ interface IDataErrorProps {
   children?: React.ReactNode;
   card?: boolean;
   className?: string;
+  // flag to use the updated DataError design
```

Comment:
Can you re-write this as

`/** Flag to use the updated DataError design */`

So it shows up on hover in VS code?